README-zh.md

```diff
@@ -0,0 +1,47 @@
+### [इसे हिंदी में पढ़ें](https://github.com/mozillascience/studyGroup/blob/gh-pages/README-hi.md)
+
+### [Leia isto em Português](https://github.com/mozillascience/studyGroup/blob/gh-pages/README-pt.md)
+
+### [Read in English](https://github.com/mozillascience/studyGroup/blob/gh-pages/README.md)
+
```

Comment:
Awesome! If you would mind adding a line to the base README.md like the ones above here, ie 

### [Read in Chinese](LINK TO README-zh.md)

That would be great! Feel free to submit a PR for that when you have time, for now I'll merge this!